app/controllers/administrative-units/new.js

```diff
@@ -144,13 +145,18 @@ export default class AdministrativeUnitsNewController extends Controller {
 
     yield Promise.all([
       administrativeUnit.validate(),
-      address.validate(),
-      contact.validate(),
-      secondaryContact.validate(),
       identifierKBO.validate(),
       identifierSharepoint.validate(),
     ]);
 
+    if (this.features.isEnabled('edit-contact-data')) {
+      yield Promise.all([
+        address.validate(),
+        contact.validate(),
+        secondaryContact.validate(),
+      ]);
+    }
+
     if (!this.hasValidationErrors) {
       const siteTypes = yield this.store.findAll('site-type');
```

Comment:
**Nitpick:** I assume this call can move into the conditional `edit-contact-data` block. (Not sure why it is called all the way up here.)